lib/VMwareWebService/MiqVimVm.rb

```diff
@@ -749,9 +749,12 @@ def addDisk(backingFile, sizeInMB, label=nil, summary=nil, thinProvisioned=false
 						    con.startConnected		= "true"
 						    con.connected			= "true"
 						end
+						mode = (dependent ? 
+							  (persistent ? VirtualDiskMode::Persistent : VirtualDiskMode::Nonpersistent ) 
+							: (persistent ? VirtualDiskMode::Independent_persistent : VirtualDiskMode::Independent_nonpersistent )
+						)
```

Comment:
This nesting of ternarys is pretty nasty, and nesting of them is generally avoided (rubocop picked it out too).  Can you make the outer case a normal if/else?